plugins/go-redisv9/hook.go

```diff
@@ -187,3 +187,13 @@ func getKey(args []interface{}) string {
 	}
 	return key
 }
+
+const maxArgsLength = 1024
```

Comment:
Hard code the limitation? I don't think this is a good practice.

Please follow this pattern, https://github.com/apache/skywalking-go/blob/main/tools/go-agent/config/agent.default.yaml

Reply:
done, please cr again 😁